source/sync/data-model/migrate-schema-partner-collection.txt

```diff
@@ -96,4 +96,132 @@ The *new* schema is the same, except we want the ``_id`` field to be a string:
 Procedure
 ---------
 
-.. include:: /includes/steps/perform-destructive-changes-synced-schema.rst
+.. procedure::
+
+   .. step:: Initialize Your Partner Collection
+
+      Since breaking changes cannot be performed directly on a synced object
+      schema, you must create a partner collection with a schema containing the
+      required changes. You must ensure that the partner collection has the same
+      data as the original collection so that newer clients can synchronize with
+      older clients.
+
+      The recommended approach to copying the data from your original collection to
+      the new partner collection is to use the :manual:`Aggregation Framework
+      </aggregation/>`.
+
+   .. step:: Create an Aggregation Pipeline
```

Comment:
In the currently-live version of the page, this is a header within a step instead of a separate step. I don't think this should be a separate step because it's _how_ we're recommending to Initialize the Partner Collection.

Reply:
Makes sense. I'll see if I can rework this.